src/main/resources/jnt_gotoManager/html/gotoManager.jsp

```diff
@@ -17,20 +17,6 @@
         </div>
     </c:if>
     <c:if test="${currentResource.workspace ne 'live'}">
-        <c:if test="${gotoType eq 'document'}">
-            <c:set var="conf" value="filemanager"/>
-            <c:set var="requiredPermission" value="fileManager"/>
-            <c:set var="label" value="label.filemanager"/>
-            <c:set var="icon" value="treepanel-files-manager-1616"/>
-            <c:set var="multisite" value="true"/>
-        </c:if>
-        <c:if test="${gotoType eq 'content'}">
-            <c:set var="conf" value="editorialcontentmanager"/>
-            <c:set var="requiredPermission" value="editorialContentManager"/>
-            <c:set var="label" value="label.contentmanager"/>
-            <c:set var="icon" value="treepanel-content-manager-1616"/>
-            <c:set var="multisite" value="true"/>
-        </c:if>
         <c:if test="${gotoType eq 'unitedContent'}">

```

Comment:
Maybe you should also update https://github.com/Jahia/default/blob/f794040d553506fff812316584e4f7600989412d/src/main/resources/META-INF/definitions.cnd#L171 ? And eventually remove the property and the if, if there's only one possible value ..

Reply:
yep, I'll fix that